backend/src/main/java/fk/prof/backend/http/BackendHttpVerticle.java

```diff
@@ -78,6 +92,11 @@ private Router setupRouting() {
     router.put(ApiPathConstants.BACKEND_PUT_ASSOCIATION)
         .handler(this::handlePutAssociation);
 
+    router.post(ApiPathConstants.BACKEND_POST_POLL)
```

Comment:
In that case let us keep it the way it is, but extract a varargs util to do this, and pump all invocation-args to it?